Components/modal/modal.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="../../css/header.css" />
+  <link rel="stylesheet" href="../../css/sidebar.css" />
+  <link rel="stylesheet" href="../../css/content.css" />
+  <link rel="stylesheet" href="../../css/divider.css" />
+  <link rel="stylesheet" href="../../css/container.css" />
+  <link rel="stylesheet" href="../../css/media-query.css" />
+  <link rel="stylesheet" href="https://storm-components.netlify.app/components.css" />
```

Comment:
Everything is good. One suggestion to import these css files in 1 file & then link to the html.

Reply:
will keep this in mind